plugin/cql/src/main/java/org/opencds/cqf/ruler/cql/JpaFhirPlatform.java

```diff
@@ -0,0 +1,43 @@
+package org.opencds.cqf.ruler.cql;
+
+import ca.uhn.fhir.rest.api.server.RequestDetails;
+
+@SuppressWarnings("unchecked")
+public class JpaFhirPlatform implements org.cqframework.fhir.api.FhirPlatform {
+
+    public ca.uhn.fhir.jpa.api.dao.DaoRegistry daoRegistry = null;
+    public RequestDetails requestDetails = null;
+
+    public JpaFhirPlatform() {}
+
+    public JpaFhirPlatform(ca.uhn.fhir.jpa.api.dao.DaoRegistry daoRegistry, RequestDetails requestDetails) {
+        this.daoRegistry = daoRegistry;
+        this.requestDetails = requestDetails;
+    }
+
+    @Override
+    public org.cqframework.fhir.api.FhirDal dal() {
+        org.cqframework.fhir.api.JpaFhirDal fhirDal = new org.cqframework.fhir.api.JpaFhirDal();
```

Comment:
Should pass the DaoRegistry and RequestDetails to the instances the platform is creating. The RequestDetails contains tenancy / manifest information, so that informs how Resources are resolved on the local server.